bftengine/cmf/execution_data.cmf

```diff
@@ -0,0 +1,22 @@
+
+Msg RequestInput 1 {
```

Comment:
Nit: maybe we can simplify`RequestInput` => just `Request` ?

Reply:
I want to emphasize that it's the input data part of the execution request since the struct itself ```struct ExecutionRequest``` contains also output data.